File: allspice/Models/Recipe.cs
namespace allspice.Models
{
  public class Recipe : IRepoItem<int>
  {
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Title { get; set; }
    public string Instructions { get; set; }
    public string Img { get; set; }
    public string Category { get; set; }
    public string CreatorId { get; set; }
    public ProfileSmall Creator { get; set; }
  }
}

File: allspice.client/src/models/Recipe.js
import { Account } from "./Account.js";

export class Recipe {
  constructor(data) {
    this.id = data.id;
    this.createdAt = data.createdAt;
    this.updatedAt = data.updatedAt;
    this.title = data.title;
    this.instructions = data.instructions;
    this.img = data.img;
    this.category = data.category;
    this.creatorId = data.creatorId;
    this.creator = new Account(data.creator);
    this.favoriteId = data.favoriteId;
  }
}
